README.md

```diff
@@ -11,6 +11,7 @@
   - [Examples](https://github.com/hashicorp/terraform-provider-kubernetes/tree/master/_examples)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 - Chat: [#terraform-providers in Kubernetes](https://kubernetes.slack.com/messages/CJY6ATQH4) ([Sign up here](http://slack.k8s.io/))
+- Roadmap: [Q3 2020](_about/ROADMAP.md)
```

Comment:
As a more general question - will we have separate roadmap documents for the helm, the operator and the kubernetes provider? If not, may make sense to centralize the efforts here and place operator work in `discovery`

Reply:
Originally I thought I would combine them into a single document, but now I think they should all have their own artifact.  It probably still makes sense to reference the operator in this roadmap for visibility.